jobsub/examples/gbl_local/config.cfg

```diff
@@ -51,10 +51,10 @@ HotpixelRunNumber	= @RunNumber@
 SkipNEvents		= 0
 
 # Output subfolder structure
-DatabasePath		= ./output/database
-HistogramPath		= ./output/histograms
-LcioPath            = ./output/lcio
-LogPath				= ./output/logs
+DatabasePath		= %(BasePath)s/output/database
```

Comment:
Did you run jobsub in the folder you created to output directory? In this case it should work. I.e. cd $EUTELESCOPE/jobsub/examples/gbl_local and there call jobsub!